wizeline/wize-q-front
# Feat - WIP-270 - Logout Confirmation Window

What does this PR do?
- Add the feature that a popup window will appear and ask for confirmation when logged in user clicks on logout button.

Where should the reviewer start?
- Please see src/components/LogoutConfirmationModal.js for the render function of the modal
- Please see changes made to src/components/App.js to add the modal to the user application section. Changes are added handler for buttons and additional attributes for the modal.

How should this be manually tested?
- After user logged in, one should see an "x" button on the top right corner. Clicking on this button should prompted a pop up window asking user for confirmation about logging out. Clicking on "x", "cancel" or anywhere outside of the popup window should resume user to use the application. Clicking on "Logout" button should successfully log the user out.

What are the relevant tickets?
https://wizeline.atlassian.net/browse/WIP-270

Screenshots (if appropriate)
<img width="676" alt="screen shot 2017-06-29 at 2 47 23 pm" src="https://user-images.githubusercontent.com/29735623/27707181-016aa7cc-5cda-11e7-8ff0-d2bcbabe81e3.png">
